### PR TITLE
Improve dev experience

### PR DESCRIPTION
**Pull request by Samywamy10**
Couple of changes, hope this is okay

- Added to `.gitignore` the files generated by `pip3 install .`
- Added local dev instructions to readme
- Created a `products_dict` out of `product_map` and `feature_map` so we're not using magic strings for `"hev"` or `"relays"`
- Added a `BulbOptions` enum for the CLI so we're not using magic numbers across the prompt for the user and the code which reads the number

### CODE REVIEW

**Comment by frawau** [comment]
Hi,

Thanks for both PR's. 

One thing though, you updated lifx-cli.py, which is simply there as an example. Why did you not update __main__.py which is the CLI that gets installed?

**Comment by Samywamy10** [comment]
Sorry I wasn't too sure what the difference between the files was. What does your development workflow look like? I added what I used in the README of this PR but keen to know how you do it

File: README.md [hunk 1]
```diff
@@ -106,6 +106,13 @@ Other things worth noting:
     - Unlike in lifxlan, set_waveform takes a dictionary with the right
       keys instead of all those parameters
 
+# Development
+To develop locally, the easiest way is to from the root directory run:
+
+```bash
+pip3 install . && python3 ./examples/lifx-cli.py
```
Comment (Djelibeybi) [inline README.md:113]:
```suggestion
pip3 install -e . && python3 ./examples/lifx-cli.py
```

Comment (frawau) [reply README.md:113]: Well.... from the project root directory

     python3 -m aiolifx

is a lot more convenient.  Clearly you need to have the dependencies installed, but I usually have the latest stable  version installed on the system I use.

The file examples/lifx-cli.py was created before I decided to have a CLI utility.

**Pull request merged**